ReactNativeClient/lib/services/InteropService_Importer_Md.js

```diff
@@ -54,21 +58,50 @@ class InteropService_Importer_Md extends InteropService_Importer_Base {
 		}
 	}
 
+	/**
+	 * Parse text for links, attempt to find local file, if found create Joplin resource
+	 * and update link accordingly.
+	 *
+	 * mdImageTagRegex matches markdown image tags.
+	 * !\[ and \] matches start of tag containing alt text.
+	 * Within this the [^\]]* matches multiple characters not in this set;
+	 * in this case the set is "]" - so match anything that is not this char.
+	 * We need this to avoid overshooting if another "]" follows after the
+	 * match.
+	 * \(.*?\) to match the part of the tag containing the link.
+	 */
+	async importLocalImages(filePath, md) {
+		const mdImageTagRegex = /!\[[^\]]*\]\((.*?)\)/g;
+		let updated = md;
+		let match;
+		while ((match = mdImageTagRegex.exec(md)) !== null) {
+			const attachmentPath = path.resolve(path.dirname(filePath), match[1]);
+			if (fs.existsSync(attachmentPath)) {
+				console.info(`Attempting to attach to note from ${filePath}, file: ${match[1]}, at position ${match.index}`);
```

Comment:
We can't have console statements in here because it's used for example in the CLI app and it would break the GUI. It's ok to leave it commented out though, so that we can reuse it when debugging.

Reply:
thanks, done

Reply:
removed